App.js

```diff
@@ -0,0 +1,506 @@
+            onPress={() => kAwake ? setkAwake(false) : setkAwake(true)}>
+            Keep Awake {kAwake && <View style={{paddingLeft: 95}}><Image
+                source={require('./icons/tick.png')}
+                style={{width: 20, height: 20, alignSelf: 'flex-end', tintColor: 'gray'}}
+              /></View>
+          }
+          </Text>
+        </View>
+        <View style={styles.menuitem}>
+          <Text
+            style={{color: nMode ? 'white' : 'black', fontSize: 21}}
+            onPress={() => isLarivaar ? setisLarivaar(false) : setisLarivaar(true)}>
+            Larivaar {isLarivaar && <View style={{paddingLeft: 132}}><Image
+                source={require('./icons/tick.png')}
+                style={{width: 20, height: 20, alignSelf: 'flex-end', tintColor: 'gray'}}
+              /></View>
+          }
+          </Text>
+        </View>
+        <View style={styles.menuitem}>
+          <Text
+            style={{color: nMode ? 'white' : 'black', fontSize: 21}}
+            onPress={() => sahajPaath ? setsahajPaath(false) : setsahajPaath(true)}>
+            Sehaj Paath <View><Text style={{width: 110, paddingLeft:30, paddingRight:10, textAlign:'right', height: 20, alignSelf: 'flex-end', fontSize:21}}>{dailyProgress}/{dailyTotal}</Text></View>
+          </Text>
+        </View>
+          { sahajPaath && <View>
+            <View style={styles.menuitem}>
+            <Text
+              style={{color: nMode ? 'white' : 'black', fontSize: 19, paddingLeft: 15}}
+              onPress={() => setAngPop(true)}>
+              Set Desired Daily Ang
+            </Text>
+            </View>
+         
+        <View style={styles.menuitem}>
+          <Text
+            style={{color: nMode ? 'white' : 'black', fontSize: 19, paddingLeft:15}}
+            onPress={() => datePicker ? setdatePicker(false) : setdatePicker(true)}>
+            Edit Samaaptee Date 
+          </Text>
+          
+          
+        </View>
+        </View> }
+        <Text>{dateSelected.toString()}</Text>
+        <View style={styles.menuitem}>
+        <CustModal />
+        </View>
+      </View>
+    </View>
+  );
+
+  const doSomething = () => {
+    isOpen ? setisOpen(false) : setisOpen(true);
+  };
+
+  const storeData = async(key, value) => {
+    try {
+      await AsyncStorage.setItem(key, value);
+    } catch (e) {
+      // saving error
+    }
+  };
+
+  
+
+  const getstoreData = async () => {
+   
+    try {
+      const value = await AsyncStorage.getItem('ang');
+      if (value !== null) {
+        setAng(value);
+        getData(value);
+        var gdt = await getsData('setdaily');
+        console.log("gdt from service is", gdt)
+        gdt && calculateTotal(gdt);
+        
+      } else {
+        storeData('ang', '1');
+        setAng('1');
+        getData(ang);
+        
+      }
+    } catch (e) {
+      // error reading value
+    }
+  };
+
+  var shabads_new = [];
+
+  useEffect(ang => {
```

Comment:
You are passing `ang` as parameter to callback function but its not used in function call or so.